README.rst

```diff
@@ -175,37 +175,80 @@ protection of Free Law Project and our users; it does not change your
 rights to use your own Contributions for any other purpose.
 
 
-Getting Set Up as a Developer
-=============================
+Development
+===========
 
-To get set up as a developer of Juriscraper, you'll want to install the code
-from git. To do that, install the dependencies and geckodriver as described above.
-Instead of installing Juriscraper via pip, do the following:
+To work on Juriscraper, clone its repository:
 
 ::
 
-    git clone https://github.com/freelawproject/juriscraper.git .
-    pip install -r requirements.txt
-    python setup.py test
+    git clone https://github.com/freelawproject/juriscraper.git
+
+Then, you can run its tests with `tox <https://tox.readthedocs.io/en/latest/>`__.
+Install tox with `uv <https://docs.astral.sh/uv/>`__ as a `tool <https://docs.astral.sh/uv/concepts/tools/>`__, adding the `tox-uv extension <https://github.com/tox-dev/tox-uv>`__:
+
+::
+
+    uv tool install tox --with tox-uv
+
+To run juriscraper’s tests for all Python versions, run:
+
+::
 
-    # run tests against multiple python versions via tox
     tox
 
-    # run network tests (on demand, not run via default command above)
-    python setup.py testnetwork
+To run tests for a single Python version, pass the environment name, such as for Python 3.13:
+
+::
+
+    tox -e py313
+
+To pass extra arguments to pytest, add them after a ``--`` separator, like:
+
+::
+
+    tox -e py313 -- --pdb
+
+Network tests
+-------------
+
+The tests in ``tests/network`` interact with PACER.
+By default, they are skipped, as they require working credentials.
+To run them, set the environment variables ``PACER_USERNAME`` and ``PACER_PASSWORD`` to your PACER credentials, for example:
+
+::
+
+    export PACER_USERNAME=the-coolest-lawyer
+    export PACER_PASSWORD=hunter2
+
+Then, run the tests as usual:
 
-You may need to also install Juriscraper locally with:
+::
+
+    tox -e py313
+
+Or, to run only the network tests:
 
 ::
 
-   pip install .
+    tox -e py313 -- tests/network
```

Comment:
Great!

Also, in line 360 we have:
`python -m unittest -v tests.local.test_DateTest.DateTest.test_various_date_extractions`

That test doesn’t exist anymore. Can we replace it with?

`tox -e py -- tests/local/test_DateTest.py::DateTest::test_date_range_creation` 
Is that the correct way to run an individual test with tox?

And in line 362 we have:

`nosetests -v --pdb tests/local/test_DateTest.py:DateTest.test_various_date_extractions`

Is there a tox equivalent? Or should we just replace it with:
`uv run nosetests -v --pdb tests/local/test_DateTest.py:DateTest.test_date_range_creation`